src/auth/AuthManager.ts

```diff
@@ -218,19 +220,19 @@ export class AuthManager {
     return await this.getLoginForm(formResponse)
   }
 
-  private async fetchUserName(): Promise<{
+  private async fetchUserName(accessToken?: string): Promise<{
     isLoggedIn: boolean
     userName: string
   }> {
-    //For VIYA we will send request on API endpoint. Which is faster then pinging SASJobExecution.
-    //For SAS9 we will send request on SASStoredProcess
```

Comment:
why are we removing these comments?

Reply:
These were added by me, when changed the url for fetching username for `SASVIYA`.
It's obvious from following lines which URLs are used for which server.

Reply:
ok